site-src/concepts/api-overview.md

```diff
@@ -117,8 +117,10 @@ modifying them in-flight.
 
 TLSRoute is for multiplexing TLS connections, discriminated via SNI. It's intended
 for where you want to use the SNI as the main routing method, and are not interested
-in properties of the higher-level protocols like HTTP.  The byte stream of the
-connection is proxied without any inspection to the backend.
+in properties of the higher-level protocols like HTTP. The byte stream of the
+connection is proxied without any inspection to the backend when using a `Passthrough` 
+TLS listener, or proxied unencrypted to the backend when using a `Terminate` TLS 
```

Comment:
Not necessarily unencrypted - some implementations (mesh ingress) may auto-reencrypt, typical N/S Gateways may encrypt to the backend through an applied BackendTLSPolicy, needs some slight rephrasing.